frontend/archive/src/main/java/org/pytorch/serve/archive/model/Manifest.java

```diff
@@ -127,8 +127,6 @@ public void setEnvelope(String envelope) {
     public enum RuntimeType {
         @SerializedName("python")
```

Comment:
@namannandan Could you please clarify if python here  and in subsequent files in java means python3?

Reply:
Given that we drop support for Python 2, would it make sense to consolidate the python runtimes into one and rely on the configuration to figure out the right executable?
https://github.com/pytorch/serve/blob/2d1bd2649583c5a505840dcb1fbf9f812ced08fd/frontend/server/src/main/java/org/pytorch/serve/util/messages/EnvironmentUtils.java#L63

Reply:
@agunapal here, `python` refers to the system default python binary and would not guarantee that it is aliased to a `python3` binary. That would depend on how the environment in which Torchserve is running has been set up. In spite of this, I think there may be use-cases where this could be helpful. Please see my comment below and let me know your thoughts.

@mreso yes, that makes sense to me.
Although, I was wondering if a use-case with the following environment would benefit from having support for both `python` and `python3` runtime options:
- `python` -> points to the system default python version (say python 3.8)
- `python3` -> points to a more recent release of python3 (say python 3.11)
This could give choice between the different python 3.x releases without having to change what `python` points to. Let me know your thoughts on if this is useful. If not, I can go ahead and remove the reference to `python3` as well.

Reply:
@namannandan yeah, I see your point, though it would look like a hacked together solution and not like a feature to me. Making runtimes configurable and extendable would probably be my preference but there is probably little gain for users out of the non-negligible effort.